istio/update.sh

```diff
@@ -15,7 +19,7 @@ while getopts ":i:v:" opt; do
   esac
 done
 
-[[ -z "${ISTIO_SHA}" ]] && ISTIO_SHA="$(grep '%global git_commit ' istio.spec | cut -d' ' -f3)"
+ISTIO_SHA=${ISTIO_SHA:-"$(grep '%global git_commit ' istio.spec | cut -d' ' -f3)"}
```

Comment:
same as above

Reply:
that's not the focus of this pr. I only changed strictly what was necessary for update.sh to succeed with my changes. Further improvements can be done in another PR.